src/homekit.cs

```diff
@@ -714,14 +714,14 @@ partial interface HMHome {
 
 		[NoTV]
 		[NoWatch]
-		[Availability (Introduced = Platform.iOS_8_0, Deprecated = Platform.iOS_9_0)]
+		[Availability (Introduced = Platform.iOS_8_0, Deprecated = Platform.iOS_9_0, Message = "Use 'ManageUsers' instead.")]
 		[Async]
 		[Export ("addUserWithCompletionHandler:")]
 		void AddUser (Action<HMUser,NSError> completion);
 
 		[NoTV]
 		[NoWatch]
-		[Availability (Introduced = Platform.iOS_8_0, Deprecated = Platform.iOS_9_0)]
+		[Availability (Introduced = Platform.iOS_8_0, Deprecated = Platform.iOS_9_0, Message = "Use 'ManageUsers' instead.")]
```

Comment:
I did not clean the beta 2 diff (like the beta 1) since there's a _typo?_ there. Did you spot it (and file a rdar) ?

Reply:
I'm sorry I don't understand what you're trying to say (: What typo? What rdar?

Reply:
The diff for beta 2 is just the addition of a message with the better API to use.

Reply:
no, read it again :)

Reply:
``` diff
-- (void)addUserWithCompletionHandler:(void (^)(HMUser * __nullable user, NSError * __nullable error))completion API_PROHIBITED(watchos, tvos) API_DEPRECATED("No longer supported.", ios(8.0, 9.0));
+- (void)addUserWithCompletionHandler:(void (^)(HMUser * __nullable user, NSError * __nullable error))completion API_UNAVAILABLE(macos, watchos, tvos) API_DEPRECATED_WITH_REPLACEMENT("-manageUsersWithCompletionHandler:", ios(8.0, 9.0));
 ```

and

``` diff
-- (void)removeUser:(HMUser *)user completionHandler:(void (^)(NSError * __nullable error))completion API_PROHIBITED(watchos, tvos) API_DEPRECATED("No longer supported.", ios(8.0, 9.0));
+- (void)removeUser:(HMUser *)user completionHandler:(void (^)(NSError * __nullable error))completion API_PROHIBITED(macos, ios, watchos, tvos) API_DEPRECATED_WITH_REPLACEMENT("-manageUsersWithCompletionHandler:", ios(8.0, 9.0));
```

What typo do you see I only see a move from `API_PROHIBITED` to `API_UNAVAILABLE` and `API_DEPRECATED` to `API_DEPRECATED_WITH_REPLACEMENT`

Reply:
@VincentDondain I think @spouliot  means

```diff
-- API_PROHIBITED(watchos, tvos) API_DEPRECATED("No longer supported.", ios(8.0, 9.0));
+- API_UNAVAILABLE(macos, watchos, tvos) API_DEPRECATED_WITH_REPLACEMENT("-manageUsersWithCompletionHandler:", ios(8.0, 9.0));
```

more specifically 

```diff
-- API_PROHIBITED(watchos, tvos)
+- API_UNAVAILABLE(macos, watchos, tvos)
```

Reply:
But HomeKit is not available yet on macOS...

Reply:
![giphy](https://user-images.githubusercontent.com/204671/42108563-878feea8-7ba0-11e8-9033-35250c5e1eeb.gif)

Reply:
![Shame](https://media.giphy.com/media/PJeKg31621Wgw/giphy.gif)

Reply:
@VincentDondain it's weird (and not symmetrical to the `add` API) so it's likely a mistake. Add it to the xcode10 tracking issue and file a rdar :) thanks!